node-runner-cli/Pipfile

```diff
@@ -4,7 +4,7 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-radixdlt_core_client = {git = "https://github.com/radixdlt/python-core-client", editable = true, ref = "1.0-beta.1"}
+core_client = {git = "https://github.com/radixdlt/python-core-client", editable = true, ref = "master"}
```

Comment:
We should not use `master` or a branch here as it's not immutable and it breaks mutability assumptions in the package resolution, and for people downloading the package and expecting it to work.

Please create a tag for a versioned release in that repo and refer to the branch here :)